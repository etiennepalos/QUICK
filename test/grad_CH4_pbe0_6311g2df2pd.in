DFT PBE0 BASIS=6-311G(2df,2pd) cutoff=1.0e-9 denserms=1.0e-8 gradient

C     -2.26648    0.08484   -0.00000
H     -1.49765    0.44914   -0.71200
H     -2.99218   -0.55975   -0.53723
H     -1.77699   -0.50080    0.80510
H     -2.79909    0.95077    0.44413
