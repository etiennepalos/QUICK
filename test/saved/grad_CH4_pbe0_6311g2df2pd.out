  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2022                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **             This software is subject to the terms of the Mozilla Public License, v. 2.0.            **
  **                                                                                                     **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Shajan, A.; Giese, T. J.; Cruzeiro, V.W.D.; Smith, J.; Miao, Y.; He, X.; Ayers, K.;      
  Brothers, E.; Goetz, A.W.; Merz, K.M. QUICK-22.03                                                        
  University of California San Diego, CA and Michigan State University, East Lansing, MI, 2022             
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Thu Sep 21 15:57:00 2023
| INPUT FILE :    ch4.in
| OUTPUT FILE:    ch4.out
| DATA FILE  :    ch4.dat
| BASIS SET PATH: /mnt/home/manathun/Work/quick-342289f/install/basis

 @ Read Job And Atom

  KEYWORD=DFT PBE0 BASIS=6-311G(2DF,2PD) CUTOFF=1.0E-9 DENSERMS=1.0E-8 GRADIENT                                                                                                                                   
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 USING LIBXC VERSION: 4.3.4
 FUNCTIONAL INFORMATION:
 NAME = PBEH (PBE0) FAMILY = HYBRID GGA KIND = EXCHANGE CORRELATION
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 STANDARD GRID = SG1
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE =    3
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-09
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-05
      GRADIENT CUTOFF  =  0.100E-06
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-07
 BASIS SET = 6-311G(2DF,2PD),  TYPE = CARTESIAN
| BASIS FILE = /mnt/home/manathun/Work/quick-342289f/install/basis/6-311G2DF2PD.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = C 
 BASIS FUNCTIONS =   35
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =   15

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    5    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    4    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =   10

 -- INPUT GEOMETRY -- :
    C          -2.2665       0.0848      -0.0000
    H          -1.4977       0.4491      -0.7120
    H          -2.9922      -0.5597      -0.5372
    H          -1.7770      -0.5008       0.8051
    H          -2.7991       0.9508       0.4441

 -- DISTANCE MATRIX -- :
              1           2           3           4           5
      1     0.00000
      2     2.09645     0.00000
      3     2.09645     3.42350     0.00000
      4     2.09646     3.42349     3.42349     0.00000
      5     2.09645     3.42349     3.42349     3.42351     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   95
 NSHELL =   31 NPRIM  =  116
 JSHELL =   31 JBASIS =   46


 @ End Reading Molecular Information 


 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =       18824
|   FINAL GRID POINTS    =       12629
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =       15213
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =       15916

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -35.147466772      ------         0.780     1        0.01      0.78      0.00    0.1805E+01  0.3855E-01  0.1724E+01
|  2    -39.026974692  0.387951E+01       1.135     2        0.01      1.13      0.00    0.6428E+00  0.2377E-01  0.7509E+00
|  3    -40.004338488  0.977364E+00       1.135     3        0.01      1.13      0.00    0.6309E+00  0.7646E-02  0.4134E+00
|  4    -40.472273868  0.467935E+00       1.133     4        0.01      1.13      0.00    0.1504E-01  0.6961E-03  0.1546E-01
|  5    -40.473120720  0.846852E-03       1.127     5        0.01      1.12      0.00    0.2375E-02  0.7833E-04  0.3487E-02
|  6    -40.473133706  0.129862E-04       1.116     6        0.01      1.11      0.00    0.5254E-03  0.1131E-04  0.2726E-03
|  7    -40.473134343  0.637271E-06       1.101     7        0.01      1.10      0.00    0.1158E-03  0.2620E-05  0.7448E-04
|  8    -40.473134388  0.445207E-07       1.077     8        0.01      1.07      0.00    0.6373E-05  0.1873E-06  0.3794E-05
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-09 ------------
|  9    -40.473134388  0.136957E-10       1.044     9        0.01      1.04      0.00    0.8379E-06  0.2352E-07  0.6721E-06
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-10 ------------
| 10    -40.473134388  -.202149E-11       1.048    10        0.01      1.04      0.00    0.1807E-06  0.5479E-08  0.3018E-06
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  10 CYCLES
| MAX ERROR = 0.180685E-06   RMS CHANGE = 0.547926E-08   MAX CHANGE = 0.301818E-06
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =     5.0000412904
 BETA ELECTRON DENSITY     =     5.0000412904
 ELECTRONIC ENERGY    =    -53.673636500
 CORE_CORE REPULSION  =     13.200502112
 TOTAL ENERGY         =    -40.473134388

 @ End Energy calculation


 @ Begin Gradient Calculation

 ANALYTICAL GRADIENT: 
----------------------------------------
 COORDINATE    XYZ            GRADIENT
----------------------------------------
    1X    -2.2664801731    -0.0000356388
    1Y     0.0848400065    -0.0000329642
    1Z    -0.0000000000     0.0000125568
    2X    -1.4976501144     0.0090524539
    2Y     0.4491400343     0.0043025304
    2Z    -0.7120000544    -0.0083627838
    3X    -2.9921802285    -0.0084828019
    3Y    -0.5597500427    -0.0075731441
    3Z    -0.5372300410    -0.0063154906
    4X    -1.7769901357     0.0057407077
    4Y    -0.5008000382    -0.0068628939
    4Z     0.8051000615     0.0094508299
    5X    -2.7990902137    -0.0062747208
    5Y     0.9507700726     0.0101664719
    5Z     0.4441300339     0.0052148876
----------------------------------------

 @ End Gradient Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     0.389573000(  2.23%)
| DFT GRID OPERATIONS =     0.057457000(  0.33%)
| TOTAL SCF TIME      =    10.703701000( 61.18%)
|       TOTAL OP TIME      =    10.652295000( 60.89%)
|             TOTAL 1e TIME      =     0.006669000(  0.04%)
|             TOTAL 2e TIME      =     8.086269000( 46.22%)
|             TOTAL EXC TIME     =     2.551965000( 14.59%)
|       TOTAL DII TIME      =     0.051245000(  0.29%)
|             TOTAL DIAG TIME    =     0.019097000(  0.11%)
| TOTAL GRADIENT TIME      =     6.321862000( 36.14%)
|       TOTAL 1e GRADIENT TIME      =     0.046865000( 0.31%)
|       TOTAL 2e GRADIENT TIME      =     5.471165000(31.27%)
|       TOTAL EXC GRADIENT TIME     =     0.796642000(  4.55%)
| TOTAL TIME          =    17.494336000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes 17.5 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Thu Sep 21 15:57:17 2023
